pkg/subctl/cmd/show_networks.go

```diff
@@ -50,14 +50,27 @@ func showNetwork(cmd *cobra.Command, args []string) {
 }
 
 func showNetworkSingleCluster(config *rest.Config) {
-	dynClient, clientSet, err := getClients(config)
-	exitOnError("Error creating clients for cluster", err)
+	submariner := getSubmarinerResource(config)
+	if submariner != nil {
+		fmt.Println("Gathering Network details from Submariner")
```

Comment:
This ends up being somewhat verbose in the resulting output. How about distinguishing the source when outputting the “Discovered network details” line? That way you’d get

```
Showing network details for cluster "cluster2":
    Discovered network details:
        Network plugin:  weave-net
        Service CIDRs:   [100.2.0.0/16]
        Cluster CIDRs:   [10.2.0.0/16]
```

when using network discovery directly, and

```
Showing network details for cluster "cluster1":
    Network details reported by Submariner:
        Network plugin:  weave-net
        Service CIDRs:   [100.1.0.0/16]
        Cluster CIDRs:   [10.1.0.0/16]
```

when using the information reported by Submariner.